docs/commands/csv2po.rst

```diff
@@ -88,34 +92,45 @@ The resultant CSV file has the following layout
 |  C     | Target Language | The msgstr or target language               |
 +--------+-----------------+---------------------------------------------+
 
+
 .. _csv2po#examples:
 
 Examples
 ========
 
-These examples demonstrate the use of csv2po::
+These examples demonstrate the use of csv2po:
+
+.. code-block:: console
 
   po2csv -P pot csv
 
-We use the :opt:`-P` option to recognise POT files found in *pot* and convert
-them to CSV files placed in *csv*::
+
+We use the :option:`-P` option to recognise POT files found in :file:`pot` and
```

Comment:
Using `:option:` without the associated `.. option::` breaks latest Sphinx.  You're only not seeing that because Sphinx is currently limited to not use the latest version. Sadly that commit that limits it provides no clarity on why.  Personally with the current task list this doesn't seem like a great fix considering that it actually is making things worse in terms of a proper fix.